tests/Domain/RegularGivingServiceTest.php

```diff
@@ -69,6 +87,105 @@ public function setUp(): void
             ->willReturn($this->donorAccount);
         $this->campaignRepositoryProphecy->findOneBySalesforceId($this->campaignId)
             ->willReturn(TestCase::someCampaign());
+        $this->regularGivingNotifierProphecy = $this->prophesize(RegularGivingNotifier::class);
+        $this->donationServiceProphecy = $this->prophesize(DonationService::class);
+        $this->entityManagerProphecy = $this->prophesize(EntityManagerInterface::class);
+
+        $this->campaignFunding = $this->createStub(CampaignFunding::class);
+    }
+
+    public function testItCreatesRegularGivingMandate(): void
+    {
+        // arrange
+        $donorId = $this->prepareDonorAccount();
+
+        /** @var Donation[] $donations*/
+        $donations = [];
+        $testCase = $this;
+        $this->donationServiceProphecy->enrollNewDonation(Argument::type(Donation::class))
+            ->will(/**
+             * @param Donation[] $args
+             */
+                function ($args) use ($testCase, &$donations) {
+                    $withdrawal = new FundingWithdrawal($testCase->campaignFunding);
+                    $withdrawal->setAmount('42.00');
+                    $args[0]->addFundingWithdrawal($withdrawal);
+
+                    $donations[] = $args[0];
+                }
+            );
+
+        $regularGivingService = $this->makeSUT(new \DateTimeImmutable('2024-11-29T05:59:59 BST'));
+
+        // act
+        $mandate = $regularGivingService->setupNewMandate(
+            $donorId,
+            Money::fromPoundsGBP(42),
+            TestCase::someCampaign(isRegularGiving: true),
+            true,
+            DayOfMonth::of(20),
+        );
+
+        // assert
+        $this->assertCount(3, $donations);
+        $this->assertSame(DonationStatus::Pending, $donations[0]->getDonationStatus());
+        $this->assertSame(DonationStatus::PreAuthorized, $donations[1]->getDonationStatus());
+        $this->assertSame(DonationStatus::PreAuthorized, $donations[2]->getDonationStatus());
+
+        $this->assertEquals(
+            new \DateTimeImmutable('2024-12-20T06:00:00 BST'),
```

Comment:
Maybe slightly confusing to specify BST & winter together?

Reply:
Thanks, fixed in e9c25989